API/src/main/java/xyz/kyngs/librelogin/api/server/ServerHandler.java

```diff
@@ -37,6 +49,7 @@ public interface ServerHandler<P, S> {
      * @param remember Whether to respect the remember last server option
      * @return An optimal lobby server, or null if there are no lobby servers
      */
+    @Deprecated
```

Comment:
What's the reason to use optional here? Deprecating an API just to move it to Optional doesn't make sense.

Reply:
Fixed this in [6824b00](https://github.com/kyngs/LibreLogin/pull/103/commits/6824b0025b2acbb09b6fd3bf686d77c1018cd577)